### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by enkog**
Milestone 6: Building with Responsive Design
-Added background images
-Added responsiveness for two break points: 1024px and 768px

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,777 @@
...
+    margin-bottom: 20px;
+  }
+
+  .fund-card:last-child {
+    display: none;
+  }
+
+  /* Categories Section */
+  .categories-section {
+    /* grid-template-columns: repeat(2, 1fr); */
```
Comment (andresporras3423) [inline css/style.css]: Remove commented code

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,777 @@
...
+
+  .update,
+  .more,
+  .info {
+    display: none;
+  }
+
+  .fund-card:last-child {
+    display: none;
+  }
```
Comment (andresporras3423) [inline css/style.css]: Group selectors with the same style definition.

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello @enkog @CornerstoneII** 
Good job with your first submission for this project :muscle:  :muscle:
Your HTML code is clean and follows the requirements :clap: :clap: :clap:  
There are some details to improve before you can go ahead:
- [x] add a screeshot to the README file
- [x] For the README file follow this template: https://github.com/microverseinc/readme-template
- [x] In the style.css file, remove commented code on line 617
- [x] In the style.css file, group selectors on lines 758 and 764.
- [x] For medium and small screen size, remove horizontal scroll.
- [x]  In medium screen size, the TNW logo should disappear:

| Yours | Original |
|:---:|:---:|
|image | image |

- [x] In medium screen size, the section after "latest funding" should be 2 columns, not three:

| Yours | Original |
|:---:|:---:|
|image | image |

-  You must define the responsive behavior for the small screen size, as the original webpage:

| Yours | Original |
|:---:|:---:|
|image | image |

Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras

**Review by MuriloRoque — APPROVED** [review]
# Status: Approved 🟢 

chuck

## Hello @enkog and @CornerstoneII 👋 

## Thank you for making all of the changes requested by the previous TSE. Congratulations, your project is approved 😄 Good luck with your next projects 🚀 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request merged**